Source/Lib/Codec/EbThreads.h

```diff
@@ -100,6 +100,8 @@ extern    EB_BOOL           alternate_groups;
 #define _GNU_SOURCE
 #include <sched.h>
 #include <pthread.h>
+#undef _GNU_SOURCE
+#undef __USE_GNU
 extern    cpu_set_t                   group_affinity;
```

Comment:
I don't understand the change here. Could you pls explain?

Reply:
There is a subtle difference with the AV1 project that this work is adapted from.

In the AVI project, EbMotionEstimation.c has a #include<math.h>, but in HEVC project, EbMotionEstimation.c does not.  This difference is causing the log2f64 naming conflict when _GNU_SOURCE is defined.

An alternative workaround would be to edit the EbMotionEstimation.c to remove the <math.h> include and replace with: extern double floor(double x).  Then the use of _GNU_SOURCE and <sched.h> in EbThreads.h would be similar to that in AV1.